### ISSUE TITLE
Glass Bangle Defect Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#50]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** :  Glass Bangle Defect Detection</br>
:red_circle: **Aim** : Create DL models to identify the defects in the bangles.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/almique/glass-bangle-defect-detection-classification</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by guptaaryan16** [comment]
Hey this is Aryan Gupta, 
GitHub Profile Link : https://github.com/guptaaryan16
Email ID : guptaaryan16@gmail.com
Participant ID (if applicable): Not available
Approach for this Project : I have been learning about Neural Networks and CNN architecture and will be using PyTorch for the implementation of the same. I relatively less experience in Deep learning so it may take some while to be implemented.

**Comment by abhisheks008** [comment]
That's a great approach @guptaaryan16. We will start working together once all the formalities are done by the SSOC team. Estimated starting date is Aug 3.

Best wishes ✨

**Comment by guptaaryan16** [comment]
Thanks Abhishek sir

**Comment by LahariBoni** [comment]
@guptaaryan16, Would you like to work on this issue?

**Comment by guptaaryan16** [comment]
Yes I want to work on this

**Comment by LahariBoni** [comment]
Assigned to you @guptaaryan16, go ahead😊

**Comment by abhisheks008** [comment]
@guptaaryan16 updates please.
